ci/infra/testrunner/tests/test_upgrade_from_4_2.py

```diff
@@ -12,6 +12,12 @@ def migrate_node(platform, checker, kubectl, role, node, regcode, option=1):
     platform.ssh_run(role, node, (f'sudo zypper migration --migration {option}'
                                   ' --no-recommends --non-interactive'
                                   ' --auto-agree-with-licenses --allow-vendor-change'))
+
+    # update to latest kubernetes package version, if any
+    k8s_major, k8s_minor, _ =  CURRENT_VERSION.split('.')
+    platform.ssh_run(role, node, 'sudo zypper ar http://download.suse.de/ibs/Devel:/CaaSP:/4.5/SLE_15_SP2/ CaasP_Devel')
```

Comment:
I wonder if, in the same spirit as we have the `CURRENT_VERSION` variable, we could have another variable for the repository where we get the packages from. This would ease up any process of migrating into another version I think. Just my 2 cents.

Feel free to ignore this comment though, this could be into another cleanup PR if needed :wink:

Reply:
Good point. That could be enhanced, definitely. I'll make a follow-up PR for that.